### DISCUSSION TITLE
How to add only private validators

### DISCUSSION THREAD

**Discussion by techie87** [0xPolygon/polygon-edge#959] [Q&A]
Hi,
I read the polygon-edge documentation. I am trying to add only known set of validators in my polygon-edge. 
Is there a way to add only specific validator address.
I understand Staking contract is deployed during genesis creation. I want to implement a whitelisted validator address. only those can stake the funds.
is that possible to re-deploy the Staking contract in the blockchain.
Please let me know

**Comment by laviniat1996** [comment]
Hi @techie87 ,

Sorry for getting back so late. You cannot choose which addresses can stake fundss in the staking contract, but you can choose a minimum and maximum number of stakers.
